tools/osbuilder/rootfs-builder/nvidia/nvidia_chroot.sh

```diff
@@ -4,7 +4,6 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-#!/bin/bash
```

Comment:
Do we need to remove this? Why?

Reply:
We have `#!/usr/bin/env bash` on the top, I am cleaning this up.